### DISCUSSION TITLE
How to turn off all feature previews by default

### DISCUSSION THREAD

**Discussion by gonmf** [community/community#58668] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Every time there is a new feature preview I have to relearn where to disable it. Is it possible to just leave all feature previews disabled by default, making them opt-in?

**Comment by sacramentosolutions** [comment]
Git does not have a built-in feature preview mechanism, so there are no specific settings to turn off feature previews in Git itself. However, if you're using a Git hosting service or a Git client tool that provides feature previews, you may be able to control their behavior through their respective configuration settings.

**Reply by gonmf** [reply]
Hello @sacramentosolutions , I meant in GitHub. Turn off all feature previews by default, instead of going to:
<img width="894" alt="Screenshot 2023-06-22 at 09 18 16" src="https://github.com/community/community/assets/5512054/f8c81c31-84b1-43e3-8b76-1d294902fb98">

**Comment by queenofcorgis** [comment, marked as answer]
Hi @gonmf, 

There isn't a capability to disable all feature previews by default currently, though you are certainly welcome to update this discussion with some more information and I can relabel it to product feedback if this is something you'd like to see. 

As a reminder, you can disable each feature preview in your settings, [learn more in our doc](https://docs.github.com/en/get-started/using-github/exploring-early-access-releases-with-feature-preview#exploring-public-preview-releases-with-feature-preview).
